control-plane/pkg/reconciler/channel/controller.go

```diff
@@ -57,5 +60,19 @@ func NewController(ctx context.Context, watcher configmap.Watcher, configs *Conf
 
 	reconciler.Resolver = resolver.NewURIResolverFromTracker(ctx, impl.Tracker)
 
+	reconciler.SecretTracker = tracker.New(impl.EnqueueKey, controller.GetTrackerLease(ctx))
+	secretinformer.Get(ctx).Informer().AddEventHandler(controller.HandleAll(reconciler.SecretTracker.OnChanged))
+
+	reconciler.ConfigMapTracker = tracker.New(impl.EnqueueKey, controller.GetTrackerLease(ctx))
```

Comment:
We can use one tracker for both:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/e0f1d1a94ab4d472d7560708dfda06cc0f29900e/control-plane/pkg/reconciler/broker/controller.go#L124-L127

Reply:
This means we cast just use one `Tracker` in our base reconciler (this could be done later)
https://github.com/knative-sandbox/eventing-kafka-broker/blob/e0f1d1a94ab4d472d7560708dfda06cc0f29900e/control-plane/pkg/reconciler/base/reconciler.go#L73-L74

Reply:
Yeah, didn't touch that part

Reply:
But, did the single tracker for configmap and secret

Reply:
cc @pierDipi